qcengine/programs/gcp.py

```diff
@@ -168,10 +167,12 @@ def build_input(
         # Need 'real' field later and that's only guaranteed for molrec
         molrec = qcel.molparse.from_schema(input_model.molecule.dict())
 
+        executable = self._defaults["name"].lower()
+
         if method == "FILE":
-            command = ["gcp", "gcp_geometry.xyz", "-local"]
+            command = [executable, "gcp_geometry.xyz", "-local"]
         else:
-            command = ["gcp", "gcp_geometry.xyz", "-level", method]
+            command = [executable, "gcp_geometry.xyz", "-level", method]
```

Comment:
For the new driver I recommend to use two dashes as shown in the help printout and in the manual page. Note that the new driver still supports one dash and the old driver supports as many dashes as you want by the logic the index function here
https://github.com/grimme-lab/gcp/blob/4be7d5acd7bc87161e0b3d89dcff81c7c761f3f7/old/main.f90#L79

The `-l` flag would be neutral for both drivers.

Reply:
fix shall appear soon